processing/src/main/java/org/apache/druid/segment/column/StringFrontCodedDictionaryEncodedColumn.java

```diff
@@ -68,12 +68,12 @@ public class StringFrontCodedDictionaryEncodedColumn implements DictionaryEncode
   private final ColumnarInts column;
   @Nullable
   private final ColumnarMultiInts multiValueColumn;
-  private final FrontCodedIndexed utf8Dictionary;
+  private final Indexed<ByteBuffer> utf8Dictionary;
```

Comment:
i think this and `ScalarStringDictionaryEncodedColumn` can now be combined but I can do that in a follow-up PR

Reply:
I did this in the latest patch.